src/converters/convert_export_declaration.ts

```diff
@@ -0,0 +1,10 @@
+import {NodePath} from '@babel/traverse';
+import {
+    ExportDeclaration,
+} from '@babel/types';
+
+export function convertEmportDeclaration(path: NodePath<ExportDeclaration>): ExportDeclaration {
+    if ((path.node as any).exportKind === null) return path.node;
```

Comment:
1. We should avoid to use `any`
2. We need to guard? I think it's unnecessary.

Reply:
This line is not necessary 🙇